iam_units/test_all.py

```diff
@@ -50,6 +50,11 @@ def test_units(unit_str, dim, new_def):
     assert registry('1 ' + unit_str).dimensionality == dim
 
 
+def test_orders_of_magnitude_14():
```

Comment:
Either use 'gh_14' so that it's clear that '14' is the number of a GitHub issue/PR, or give the URL instead of the number in the function name, or even omit both since 'blame' can tell us when this was added.

Reply:
reference omitted for simplicity